packages/common/src/utils/DateUtil.ts

```diff
@@ -21,14 +24,16 @@ dayjs.updateLocale('en', {
     y: '1 yr',
     yy: '%d yr'
   }
-})
+}
+
+dayjs.locale('en-custom', localeObject)
 
 export const DateUtil = {
   getYear(date: string = new Date().toISOString()) {
     return dayjs(date).year()
   },
 
   getRelativeDateFromNow(date: string | number) {
-    return dayjs(date).fromNow(true)
+    return dayjs(date).locale('en-custom').fromNow(true)
```

Comment:
This is how I would propose to fix the global pollution of the dayjs en locale. Happy to try other solutions as well but I think this clean enough